include/exometer.hrl

```diff
@@ -36,3 +36,11 @@
 -record(exometer_shared, {
           name
          }).
+
+-ifdef(OTP_RELEASE). %% this implies 21 or higher
```

Comment:
Where is this `OTP_RELEASE` set? I would expect it to be set somewhere in rebar.config when we detect we run on Erlang/OTP 21 or higher.